pennylane/devices/null_qubit.py

```diff
@@ -305,7 +357,7 @@ def new_shots_stopping_condition(op):
 
         updated_values = {}
         if execution_config.gradient_method in ["best", "adjoint"]:
-            updated_values["gradient_method"] = "device"
+            updated_values["gradient_method"] = "device"  # This would be the line to change
```

Comment:
```suggestion
            updated_values["gradient_method"] = "device"
```

Reply:
Good catch, I must've accidentally left that comment in there while debugging.